### PR TITLE
double-conversion@3.3.0

### PR DESCRIPTION
**Pull request by BYVoid**

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (double-conversion) have been updated in this PR. Please review the changes.

File: modules/double-conversion/3.3.0/presubmit.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  bazel:
+    - 7.x
+    - 6.x
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_targets:
+      - "@double-conversiondouble-conversion"
```
Comment (meteorcloudy) [inline modules/double-conversion/3.3.0/presubmit.yml:17]: Just `@double-conversion`?

File: modules/double-conversion/3.3.0/presubmit.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      - windows
+    bazel:
+      - 7.x
+      - 6.x
+  tasks:
+    run_test_module:
+      name: Run test module
+      platform: ${{ platform }}
+      bazel: ${{ bazel }}
+      build_targets:
```
Comment (meteorcloudy) [inline modules/double-conversion/3.3.0/presubmit.yml:35]: Missing actual target.

**Review by meteorcloudy — CHANGES_REQUESTED** [review]

File: modules/double-conversion/3.3.0/presubmit.yml (hunk 1 above)
Comment (BYVoid) [reply modules/double-conversion/3.3.0/presubmit.yml]: Thanks for catching the the typo.

File: modules/double-conversion/3.3.0/presubmit.yml (hunk 2 above)
Comment (BYVoid) [reply modules/double-conversion/3.3.0/presubmit.yml]: thanks.

**Review by meteorcloudy — APPROVED** [review]

**Pull request merged**